sdk/storage/azure-storage-files-shares/vcpkg/Config.cmake.in

```diff
@@ -0,0 +1,8 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(azure-storage-common-cpp)
```

Comment:
Why the -cpp at the end?

Reply:
It is something for us to choose, we can get rid of it. But, to an extent, this is somewhat common. See RED - ports that are not in this PR, YELLOW are. I think, `-cpp` prefix is a good thing to prevent collision between libs from `azure-sdk-for-cpp` and others (for example, `-c`). But we can remove.
![image](https://user-images.githubusercontent.com/41349689/99864656-80ec8500-2b59-11eb-81fc-58e74d9e478d.png)

Reply:
FWIW, Google and AWS do also have `-cpp` suffix:
![image](https://user-images.githubusercontent.com/41349689/99865081-459f8580-2b5c-11eb-906e-d296365475a5.png)

Reply:
It makes sense if Vcpkg can be used for something else than cpp..  Otherwise, it should be understood that it is all cpp.
libcurl and cmocka don't follow this (libcurl-cpp, cmocka-cpp)

Maybe these libraries use it because they have other languages support right?  (libcurl is only cpp, for instance)

I think the -cpp is fine

Reply:
One correction, libcurl is C, not Cpp :)

Reply:
Yes, my bad! lol
Then that's the quick reason right?   Vcpkg contains both C and CPP packages. We want to add the -cpp to make it clear for users about it when using `./vcpkg search`